### ISSUE TITLE
Subtle inconsistency?

### ISSUE THREAD

**Issue by fohly** [memononen/SDF#4]
Line 290 in sdf.h compares tdist[kn] to tdist[k], while all other similar blocks compare it to pd.
See https://github.com/memononen/SDF/blob/1ef5b4e4feb60ea84e47c01425de3edcf39bd83c/src/sdf.h#L290

Is this a mistake or is there a reason behind this?
Although I think that this doesn't make much of a difference.

**Comment by memononen** [comment]
You're right, it should be pd. There are some cases which do not work well,
i wonder if this is the problem.

ti 14. toukok. 2019 klo 11.27 fohly <notifications@github.com> kirjoitti:
